src/router/validation/schemas/fhir.schema.v3.json

```diff
@@ -4793,15 +4793,14 @@
             },
             "category": {
               "description": "Category of the identified substance.",
-              "enum": [
```

Comment:
Are there other enums in STU3 that are like this? if this is a large task we can put it in our backlog

Reply:
there may be more, It's hard to spot them just by looking. Scripting can aid in finding out. I'll add an item to the backlog.